README.md

```diff
@@ -1,2 +1,8 @@
 # GitHubAiAPI
 This is Example to test AI plugin for automatic code review
+
+# Resources
+https://docs.github.com/en/rest/quickstart?apiVersion=2022-11-28
```

Comment:
The 'Resources' section could be more descriptive. Instead of just providing a link, briefly explain what the resource is about.